Server/modules/src/systeminfo.py

```diff
@@ -0,0 +1,15 @@
+import System.DateTime as DateTime
+import System.Environment as Env
+import System.Diagnostics as Diagnostics
+
+def getSystemInfo():
+    summary = "\nHost: \t{0}\n".format(Env.MachineName)
+    summary += "OS: \t{0} {1}\n".format(Env.OSVersion.Platform, Diagnostics.FileVersionInfo.GetVersionInfo(Env.SystemDirectory + "\\kernel32.dll").ProductVersion)
+    summary += "64-Bit: {0}\n".format(Env.Is64BitOperatingSystem)
+    summary += "Domain: {0}\n".format(Env.UserDomainName)
+    summary += "User: \t{0}\n".format(Env.UserName)
+    summary += "Date: \t{0}\n".format(DateTime.Now.ToString())
+    print summary
+
+getSystemInfo()
```

Comment:
Maybe you can move the `print` outside the method (like this https://github.com/byt3bl33d3r/SILENTTRINITY/blob/master/Server/modules/src/shell.py)

```python
def getSystemInfo():
    summary = "\nHost: \t{0}\n".format(Env.MachineName)
    summary += "OS: \t{0} {1}\n".format(Env.OSVersion.Platform, Diagnostics.FileVersionInfo.GetVersionInfo(Env.SystemDirectory + "\\kernel32.dll").ProductVersion)
    summary += "64-Bit: {0}\n".format(Env.Is64BitOperatingSystem)
    summary += "Domain: {0}\n".format(Env.UserDomainName)
    summary += "User: \t{0}\n".format(Env.UserName)
    summary += "Date: \t{0}\n".format(DateTime.Now.ToString())
    return summary
print getSystemInfo() 
```